### PR TITLE
Create a tox docs environment

### PR DESCRIPTION
**Pull request by s-t-e-v-e-n-k**
To make it easier to build documentation locally, add a new tox
environment that installs our requirements and runs Sphinx.

### CODE REVIEW

**Comment by sfdye** [comment]
LTGM. Can you also update the contribution docs on how to use this locally?

**Comment by s-t-e-v-e-n-k** [comment]
@sfdye Good point, done

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,3 +115,9 @@ tox -epy36,flake8,black,isort
 pip install -r requirements.txt
 sphinx-build doc build
 ```
+
+If you use tox:
+
+```
+tox -edocs
```
Comment (sfdye) [inline CONTRIBUTING.md]:
```suggestion
tox -e docs
```
I suppose?

Comment (s-t-e-v-e-n-k) [reply CONTRIBUTING.md]: Both work fine, and the no space matches what I added before :-P

Comment (sfdye) [reply CONTRIBUTING.md]: I see, I just notice there is a missing backtick here:
https://github.com/PyGithub/PyGithub/blob/3b9cfc6fa5ce5e317888f5ba89cf0c458304b8b1/CONTRIBUTING.md#L107

Comment (sfdye) [reply CONTRIBUTING.md]: which will cause the rest not rendering correctly

Comment (s-t-e-v-e-n-k) [reply CONTRIBUTING.md]: Ahh, let's fix that properly in this commit!

Comment (s-t-e-v-e-n-k) [reply CONTRIBUTING.md]: Fixed by force-push

**Pull request merged**